### PR TITLE
MODORDSTOR-50 Use sample data in unit tests

### PR DESCRIPTION
**Pull request by grabsefx**
https://issues.folio.org/browse/MODORDSTOR-50

## Purpose
* In order to minimize the amount of `sample/test/example` data that needs to be maintained, data in {{src/test/resources}} should be removed and used the sample data instead.

## Approach
* Example data deleted
* Unit tests updated
* Sample data content sorted in alphabetical order (except `id` field) for better readability
* Removed metadata from PO an POLine samples

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
   "itemId": "522a501a-56b5-48d9-b28a-3a8f02482d97",
   "locationId": "53cf956f-c1df-410b-8bea-27f712cca7c0",
-  "poLineId": "d471d766-8dbb-4609-999a-02681dea6c22",
+  "poLineId": "90fb53f2-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "receivedDate": "2018-10-10T00:00:00.000Z",
```
Comment (piotr-kalashuk) [inline src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json]: The `receivingStatus` is `Expected` but `receivedDate` is specified...

File: src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
   "itemId": "15447c41-bc6a-4600-96a4-a1ce7f44c62a",
   "locationId": "fcd64ce1-6995-48f0-840e-89ffa2288371",
-  "poLineId": "2fe6c2dd-3700-4a53-a624-1159cfd7f8ce",
+  "poLineId": "50fb6ae0-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "receivedDate": "2019-01-11T00:00:00.000Z",
   "receivingStatus": "Expected",
```
Comment (piotr-kalashuk) [inline src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json]: Better to set `receivingStatus` to `Received`

File: src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
   "itemId": "522a501a-56b5-48d9-b28a-3a8f02482d97",
   "locationId": "53cf956f-c1df-410b-8bea-27f712cca7c0",
-  "poLineId": "d471d766-8dbb-4609-999a-02681dea6c22",
+  "poLineId": "90fb53f2-cdf1-11e8-a8d5-f2801f1b9fd1",
```
Comment (piotr-kalashuk) [inline src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json]: This PO Line `90fb53f2-cdf1-11e8-a8d5-f2801f1b9fd1` has `"quantityPhysical": 6` but only one piece is available. Taking into account that you had to add some pieces for unit tests probably it worth to align with that quantity or use, for example, `b0fb580c-cdf1-11e8-a8d5-f2801f1b9fd1` as `poLineId`

File: src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json [hunk 4]
```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
```
Comment (piotr-kalashuk) [inline src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json]: PO Line has `"orderFormat": "Electronic Resource"` so please change `format` to `Electronic`

File: src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json (hunk 3 above)
Comment (grabsefx) [reply src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json]: PO line id updated

File: src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json (hunk 4 above)
Comment (grabsefx) [reply src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json]: format updated

File: src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json (hunk 2 above)
Comment (grabsefx) [reply src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json]: `receivingStatus` updated

File: src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json (hunk 1 above)
Comment (grabsefx) [reply src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json]: removed `receivedDate` field from sample

**Pull request merged**